unstructured_ingest/v2/processes/connectors/milvus.py

```diff
@@ -175,11 +175,18 @@ class MilvusUploader(Uploader):
 
     @DestinationConnectionError.wrap
     def precheck(self):
-        with self.get_client() as client:
-            if not client.has_collection(self.upload_config.collection_name):
-                raise DestinationConnectionError(
-                    f"Collection '{self.upload_config.collection_name}' does not exist"
-                )
+        from pymilvus import MilvusException
+
+        try:
+            with self.get_client() as client:
+                if not client.has_collection(self.upload_config.collection_name):
+                    raise DestinationConnectionError(
+                        f"Collection '{self.upload_config.collection_name}' does not exist"
+                    )
+        except MilvusException as milvus_exception:
+            raise DestinationConnectionError(
+                f"failed to precheck Milvus: {str(milvus_exception.message)}"
```

Comment:
We have to use `str(milvus_exception.message)`, otherwise just for `str(milvus_exception)` was getting

> Failed to redact: 2024-12-05 17:16:18,380 MainProcess ERROR    Step precheck failure: MilvusUploader: [DestinationConnectionError] failed to precheck milvus: <MilvusException: (code=<bound method _InactiveRpcError.code of <_InactiveRpcError of RPC that terminated with:
        status = StatusCode.UNKNOWN
        details = "Token not found or unauthorized: The provided token does not exist in our records, or you do not have the necessary permissions to use this token. Please ensure you have the correct token."
        debug_error_string = "UNKNOWN:Error received from peer  {created_time:"2024-12-05T17:16:18.378661811+01:00", grpc_status:2, grpc_message:"Token not found or unauthorized: The provided token does not exist in our records, or you do not have the necessary permissions to use this token. Please ensure you have the correct token."}"